frontend/package.json

```diff
@@ -18,23 +18,28 @@
     "@bufbuild/protoc-gen-es": "^1.10.0",
     "@trustwallet/wallet-core": "^4.1.4",
     "@types/react-router-dom": "^5.3.3",
+    "7z-wasm": "^1.1.0",
     "buffer": "^6.0.3",
     "crypto-browserify": "^3.12.0",
     "i18next": "^23.12.2",
     "i18next-browser-languagedetector": "^8.0.0",
     "i18next-xhr-backend": "^3.2.2",
+    "lottie-react": "^2.4.0",
```

Comment:
This package is not being maintained anymore. Do we have any alternative?

https://snyk.io/advisor/npm-package/lottie-react

Reply:
Don't need to find alternatives. This is still widely used package, you can see the weekly download is huge, it supports React18, that means that don't need to care about its maintenance for now.